pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1644,21 +1633,6 @@ func (configgen *ConfigGeneratorImpl) onVirtualOutboundListener(
 // that the health check ports are distinct from the service ports.
 func buildSidecarInboundMgmtListeners(node *model.Proxy, push *model.PushContext, managementPorts model.PortList, managementIP string) []*xdsapi.Listener {
 	listeners := make([]*xdsapi.Listener, 0, len(managementPorts))
-
-	if managementIP == "" {
-		managementIP = "127.0.0.1"
-		addr := net.ParseIP(node.IPAddresses[0])
-		if addr != nil && addr.To4() == nil {
-			managementIP = "::1"
-		}
-	}
```

Comment:
why this change?

Reply:
managementIP  can not be "",  which is guaranteed here
```
	for _, ip := range node.IPAddresses {
		managementPorts := push.ManagementPorts(ip)
		management := buildSidecarInboundMgmtListeners(node, push, managementPorts, ip)
		mgmtListeners = append(mgmtListeners, management...)
	}
```